crates/vsock/src/main.rs

```diff
@@ -35,6 +35,10 @@ struct VsockArgs {
     /// Unix socket to which a host-side application connects to.
     #[clap(long)]
     uds_path: String,
+
+    /// The size of the buffer used for the TX virtqueue
+    #[clap(long, default_value_t = 64 * 1024)]
```

Comment:
`64 * 1024` seems to be a magic number, can this be extracted to a constant?

Reply:
This value is only used here. I don't know if it is necessary to extract it

Reply:
I don't think this is true, for example some cases are:
- https://github.com/rust-vmm/vhost-device/pull/324/files/addf211be335b3787af5b6512805ca0ace146601#diff-bf42ff27bba7da91addf730f13ba53e10b95e23f5bff62b18cf7d14d8e643fe0R131
- https://github.com/rust-vmm/vhost-device/pull/324/files/addf211be335b3787af5b6512805ca0ace146601#diff-bf42ff27bba7da91addf730f13ba53e10b95e23f5bff62b18cf7d14d8e643fe0R140
- https://github.com/rust-vmm/vhost-device/pull/324/files/addf211be335b3787af5b6512805ca0ace146601#diff-bf42ff27bba7da91addf730f13ba53e10b95e23f5bff62b18cf7d14d8e643fe0R149

Reply:
@JonathanWoollett-Light all of them are tests, right?

I do not have a strong opinion on this, in fact I would maybe even use values other than the default in tests.

Reply:
@stefano-garzarella Yes these are all in tests.

I think it may be preferable to use a constant both in `#[clap(long, default_value_t = 64 * 1024)]` and in the tests if the same value is used.

But ultimately I do not have a strong opinion on this either.